packages/generate-loader/types/Yeoman.ts

```diff
@@ -0,0 +1,5 @@
+export interface IYeoman {
```

Comment:
`any` makes use of typescript useless. Debug the types yourself to find the correct ones

Reply:
The methods present in this interface are typed precisely and not using `any`. Where exactly are you pointing?

Reply:
It was `any` before, maybe I reviewed an old commit